packages/@lwc/module-resolver/scripts/test/matchers/to-throw-error-with-code.ts

```diff
@@ -5,11 +5,11 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 
-import diff from 'jest-diff';
-import MatcherUtils = jest.MatcherUtils;
+import diff from '@vitest/utils/diff';
+import type { MatcherState } from '@vitest/expect';
```

Comment:
These packages both already come from `vitest`, but it makes me nervous not to have these explicitly declared in the `package.json`. Let's add them.